proto/storage/image.proto

```diff
@@ -104,6 +108,9 @@ message ImageSignatureVerificationResult {
     Status status                               = 3;
     // description is set in the case of an error with the specific error's message. Otherwise, this will not be set.
     string description                          = 4 ;
+    // The full image name that the signature references. Since images can come from different locations, this will
+    // ensure we can verify the matching location as well.
+    repeated string verified_image_references = 5;
```

Comment:
Can we move this field inside `ImageSignatureVerificationData`?

Reply:
Makes sense.

I'll change the type from `repeated string` to something else, as I imagine currently we'd like to have the coupling between verified image names and the signature integration ID.

Reason for this is that we currently in the policy have the user select a signature integration ID with which they expect the image's signature to be verified with.
Imagining we have multiple signature integrations, and each verifies different image names, we would lose this relationship there and potentially not create an alert for a specific image name and a specific signature integration ID.